cloudbridge/providers/gcp/provider.py

```diff
@@ -230,6 +230,8 @@ def __init__(self, config):
 
         if self.credentials_dict and 'project_id' in self.credentials_dict:
             self.project_name = self.credentials_dict['project_id']
+        elif self._get_config_value("project_id"):
+            self.project_name = self._get_config_value("project_id")
```

Comment:
We can also leave this in, and perhaps simplify it as follows:
```
   self.project_name = self._get_config_value("project_id", os.getenv('GCP_PROJECT_NAME'))
```
If we do this, the only issue is that there will be 3 sources for project_id to come in from: the credentials_dict, a direct config value or the os environment, in that order. Any opinions @afgane @almahmoud 

Also, there's the question of why the ENV var and variable is called PROJECT_NAME, but the gcp nomenclature is project_id.

Reply:
I guess if we follow your previous comment, there will be only two sources for `project_id`, and that makes sense where one is directly passed, and one read from an environment variable.